include/dev/RedundantADC.hpp

```diff
@@ -0,0 +1,59 @@
+#pragma once
+
+#include <EVT/io/ADC.hpp>
+
+namespace IO = EVT::core::IO;
+
+namespace HIB::DEV {
+
+/**
+ * This class allows processing readings from redundant ADCs and checking for errors
+ */
+
+class RedundantADC {
+public:
+    /**
+     * Enum class representing the status of RedundantADC processing.
+     */
+    enum Status {
```

Comment:
I think `OFF_BY_ONE_ERROR` and `MARGIN_ERROR` could use better names, but I'm not sure what would be better. See if you can come up with anything